src/simpleAuth.ts

```diff
@@ -1,8 +1,10 @@
+
```

Comment:
Why blank lines on 1 and 6?

Reply:
Don't know on one.  Accident.  The space at 6 is to delineate common packages from app specific packages